lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @file
+ * Code for the dosomething_rogue feature.
+ */
+
+define('ROGUE_API_URL', variable_get('dosomething_rogue_url', 'https://qa-rogue.dosomething.org'));
```

Comment:
set this default to `roge.dev` or whatever the local uri is, so we don't all connect to the qa instance locally